domain/consensus/processes/dagtraversalmanager/dagtraversalmanager_test.go

```diff
@@ -111,3 +208,52 @@ func TestLowestChainBlockAboveOrEqualToBlueScore(t *testing.T) {
 		checkExpectedBlock(tipHash, 10, blueScore11BlockHash)
 	})
 }
+
+func createAChainDag(genesisHash *externalapi.DomainHash, tc testapi.TestConsensus) (*externalapi.DomainHash, error) {
+	block := genesisHash
+	var err error
+	for i := 0; i < 5; i++ {
+		block, _, err = tc.AddBlock([]*externalapi.DomainHash{block}, nil, nil)
+		if err != nil {
+			return nil, err
+		}
+	}
+	return block, nil
+
```

Comment:
This empty line is redundant, but you should add an empty line after the }

I'm not commenting about any more empty lines, just go over the code and clear all of them.

There's a use for empty lines - to separate two distinct blocks of code. These should stay. But just empty lines before or after a { or } shouldn't be there.